packages/ketcher-react/src/script/ui/views/toolbars/TopToolbar/IconButton.tsx

```diff
@@ -35,16 +35,19 @@ const Button = styled(MuiButton)`
   width: 28px;
 
   &:hover {
-    color: #005662;
+    color: #006775;
     transform: none;
     box-shadow: none;
     transition: none;
     background: none;
   }
 
   &:active {
-    background-color: #005662;
+    background-color: #006775;
     color: white;
+    &:hover {
+      background-color: #188794;
+    }
   }
```

Comment:
Should use '@color-spec-button-primary-hover' here

Reply:
fixed